File: src/test/fixtures/python/coherence/unrelatedClasses.py
# decision: two classes with no shared inheritance, no type cross-reference, and no shared
# naming affix (Logger/HttpClient) - a real grab-bag of unrelated domains that should be
# flagged even though each class is individually small and internally cohesive.


class Logger:
    def __init__(self, prefix: str) -> None:
        self.prefix = prefix

    def log(self, message: str) -> None:
        print(self.prefix, message)


class HttpClient:
    def __init__(self, base_url: str) -> None:
        self.base_url = base_url

    def get(self, path: str) -> bytes:
        return b""

File: src/test/fixtures/kotlin/coherence/UnrelatedClasses.kt
// decision: two classes with no shared inheritance, no type cross-reference, and no shared
// naming affix - a real grab-bag that should be flagged even though each class is small and
// internally cohesive.

class Logger {
    fun log(message: String) {
        println(message)
    }
}

class HttpClient {
    fun get(path: String): String {
        return path
    }
}
